docs/metadata.md

```diff
@@ -113,3 +114,14 @@ method.
 ```js
 client.metadata.getTemplates('enterprise', callback);
 ```
+
+Delete Metadate Template
+------------------------
+
+To delete a metadata template call the
+[`metadata.deleteTemplate(scope, template, callback)`](http://opensoure.box.com/box-node-sdk/Metadata.html#deleteTemplate)
+method with the template scope and template name.
+
+```js
+client.metadata.deleteMetadata('enterprise', 'testtemplate', callback);
```

Comment:
This example needs to be updated to use the correct method name from above — should be `deleteTemplate()`